custom_components/google_home/models.py

```diff
@@ -40,26 +51,26 @@ def __init__(
         self._timers: List[GoogleHomeTimer] = []
         self._alarms: List[GoogleHomeAlarm] = []
 
-    def set_alarms(self, alarms: List[Dict[str, Any]]) -> None:
+    def set_alarms(self, alarms: List[AlarmJsonDict]) -> None:
         """Stores alarms as GoogleHomeAlarm objects"""
         self._alarms = [
             GoogleHomeAlarm(
-                alarm_id=alarm[ID],
-                fire_time=alarm[FIRE_TIME],
-                label=alarm.get(LABEL),
-                recurrence=alarm.get(RECURRENCE),
+                alarm_id=alarm["id"],
+                fire_time=alarm["fire_time"],
+                label=alarm.get("label"),
+                recurrence=alarm.get("recurrence"),
```

Comment:
Why are we back at hardcoding the values? What if one day google device to change a parameter name from `fire_time` to `fire_time_ms` or something else? We would need to go through multiple places with hardcoded `fire_time` to manually replace those? Is it not better to have one place(as const) where all these hardcoded values are stored and maintained? 🤔

Reply:
`TypedDict` only allows string literals here. But it verifies this strings match fields in corresponding `TypedDict`.